tests/components/ring/conftest.py

```diff
@@ -25,20 +25,16 @@ def mock_setup_entry() -> Generator[AsyncMock, None, None]:
 
 
 @pytest.fixture
-def mock_ring_auth():
-    """Mock ring_doorbell.Auth."""
-    with patch("ring_doorbell.Auth", autospec=True) as mock_ring_auth:
-        mock_ring_auth.return_value.fetch_token.return_value = {
-            "access_token": "mock-token"
-        }
-        yield mock_ring_auth.return_value
+def mock_ring_auth_client():
+    """Mock ring_doorbell client."""
 
+    mock_auth = create_autospec(ring_doorbell.Auth)
+    mock_auth.return_value.fetch_token.return_value = {"access_token": "mock-token"}
 
-@pytest.fixture
-def mock_ring():
-    """Mock ring_doorbell.Ring."""
-    with patch("ring_doorbell.Ring", autospec=True) as mock_ring:
-        yield mock_ring.return_value
+    mock_ring_auth_client = MockRingAuthClient(mock_auth.return_value)
+    mock_auth.return_value.query.side_effect = mock_ring_auth_client._get_query_response
+    with patch("ring_doorbell.Auth", new=mock_auth):
```

Comment:
You should mock this at the place where the library is imported/created

Reply:
Fixed in https://github.com/home-assistant/core/pull/113397